com.unity.formats.alembic/Third Party Notices.md

```diff
@@ -1,43 +1,24 @@
 This package contains third-party software components governed by the license(s) indicated below:
 ---------
 
-# OpenEXR
+Component Name: OpenEXR
+License Type: Modified BSD License
+Copyright © 2006-2019 Series of LF Projects, LLC
+https://www.openexr.com
```

Comment:
All these lines are currently rendered on a single line (this is the way markdown works).
If we want to have line breaks, two solutions:
- Add an empty line between lines (it will make separate paragraphs), OR
- Add `<br />` tags (it will make it more compact, like in text view).
Usually I'm adding the `<br />` tags at the beginning of the lines, it's more convenient to view the structure.